app/modules/parsing/graph_construction/parsing_helper.py

```
@@ -1831,6 +1894,87 @@ async def _create_git_worktree(
+
+            # Remove existing worktree if it exists
+            if worktree_path.exists():
+                logger.info(
+                    f"Removing existing worktree directory: {worktree_path}"
+                )
+                shutil.rmtree(worktree_path, ignore_errors=True)
+                # Also remove from git's worktree registry
+                try:
+                    bare_repo.git.worktree("prune")
+                except Exception:
+                    pass
```

Ruff (0.15.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)